compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -435,6 +431,11 @@ convertTypeDef env o@(ATyCon t) = withRange (convNameLoc t) $ if
     | isClassTyCon t
     -> convertClassDef env t
 
+    -- Type synonyms get expanded out during conversion (see 'convertType'), but we also
+    -- convert the synonyms we can so that we can expose them via data-dependencies.
+    | isTypeSynonymTyCon t
+    -> convertTypeSynonym env t
```

Comment:
Is there a technical reason for moving this further down or is it just a stylistic thing? I'm fine either way but would like to understand the former if that's the case.

Reply:
I moved it down below typeclasses and constraint tuples before realizing it doesn't make a difference anyway.

Reply:
Thanks for the explanation.